src/userreducers.ts

```diff
@@ -0,0 +1,37 @@
+import { Reducer } from "redux";
+import { IUserState, UserActionTypes } from "./facility";
+
+import {UserActions} from "./user";
+
+const initialUserState: IUserState = {
+    user: {
+        userid:1,
+        globus_email:'',
+        username:'jdoe'
+    },
+    userLoading: false
+};
+
+export const userReducer: Reducer<IUserState, UserActions> = (
+    state = initialUserState,
+    action
+) => {
+    switch (action.type) {
+        case UserActionTypes.GETINFO: {
+            return {
+                ...state,
+                user: action.user,
+                userLoading: false
+            };
+        }
+        // case UserActionTypes.LOADING: {
```

Comment:
Why is this commented out? The call could take take, and this should work as expected.

Reply:
have removed all actions that are currently unimplemented and commented out